### DISCUSSION TITLE
Turn off Copilot for a single project?

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#32254] [Copilot Conversations] [answered]
I am using VSCode for both my hobby projects and my college course. I am not allowed to use GitHub Copilot for my college course. Can I turn it off for the project for the course or can I only disable it globally?

**Comment by augustoaraujoo** [comment]
you can disable in vscode extensions, just search for github copilot and click to disable

**Reply by PythonCoderAS** [reply]
This disables in all projects, not just current project

**Reply by augustoaraujoo** [reply]
below you can see the emoji from github i think by clicking on it you have some options

**Reply by augustoaraujoo** [reply]
in your vscode

**Reply by PythonCoderAS** [reply]
?

**Reply by augustoaraujoo** [reply]
![image](https://user-images.githubusercontent.com/81258318/189366267-ef3e8df2-7302-4f88-8bb9-a7910d024c02.png)
can y see my mouse?

**Reply by PythonCoderAS** [reply]
This says "Disable Globally". I **do not want to disable globally**. _only for a single project_.

**Comment by librity** [comment, marked as answer]
You can disable it for your current workspace:
`Extentsions` => `Github Copilot` => `Disable Dropdown` => `Disable (Workspace)`
![Screenshot from 2022-09-09 11-11-43](https://user-images.githubusercontent.com/19496542/189370613-b5c1b729-b450-4d65-81cd-11b88336abf3.png)
